examples/backend-adapters/README.md

```diff
@@ -5,13 +5,14 @@
 </a>
 
 This example contains a vite-react app in the [/client](./client/) directory, as
-well as multiple backend servers in the [/server](./server/) directory. Servers
-that are included here are:
+well as multiple backend servers in the [/server](./server/) and
+[/nitro](./nitro/) directories. Servers that are included here are:
```

Comment:
Would be kind of nice to adjust the nesting here so that there are just `client` and `server` folders and inside the `server` folder there would be `nitro` and `others`

Reply:
Problem is nitro requires that routes/api folder and (I think) a nitro config. I could just replace nitro with a base node.js h3 app though

Reply:
The nitro config isn't required. But a base h3 app (with listhen server?) could be fine also.

Reply:
> The nitro config isn't required. But a base h3 app (with listhen server?) could be fine also.

Cool, I'll do a listhen server then, we still have the nitro "setup" in the docs

Reply:
Hmm @danielroe now the issue I was telling you about yesterday is back... It's matching the /api route to /api/uploadthing instead of the actual uploadthing routes...

![CleanShot 2023-09-29 at 09 39 36@2x](https://github.com/pingdotgg/uploadthing/assets/51714798/b34b2863-192f-4483-a3bf-7f6b029b2020)

I pushed what I got if you can spot any mistakes...